src/Display/Tabs/Tabs.tsx

```diff
@@ -111,12 +111,14 @@ type Tabs = React.FunctionComponent<Props> & {
 
 Tabs.Pane = TabPane;
 
-interface Props {
+export interface Props {
   children: React.ReactNode;
   variant?: Variant;
   alignment?: Alignment;
   theme?: Theme;
```

Comment:
Could we just copy & paste the previous descriptions for `theme` `alignment` and `variant` to here? 🙏

Reply:
Fixed